server/lib/nl/fulfillment/comparison.py

```diff
@@ -23,17 +23,9 @@
 from server.lib.nl.fulfillment.types import ChartVars
 from server.lib.nl.fulfillment.types import PopulateState
 from server.lib.nl.fulfillment.utils import add_chart_to_utterance
+from server.lib.nl.fulfillment.utils import get_places_as_string
 
-_MISSING_PLACE_MSG = 'Data for "{missing_places}" is currently unavailable. Nonetheless, take a look at the following insights on "{exist_places}"'
-
-
-def _get_place_list_str(places: List[str]):
-  if len(places) < 1:
-    return ''
-  elif len(places) == 1:
-    return places[0]
-  else:
-    return ', '.join(places[0:len(places) - 1]) + f' and {places[-1]}'
+_MISSING_PLACE_MSG = 'Data for "{missing_places}" is currently unavailable. See the following statistics on the other places'
```

Comment:
nit: Should we move this to `commentary.py`, so we kinda have all the msgs in one place in case we want to reuse?

Reply:
good idea, done